### ISSUE TITLE
weight decay of last layer 

### ISSUE THREAD

**Issue by forseeking** [deepinsight/insightface#603]
When the number of ID is large say 1000k, it became very difficult to converge using weight_decay 0.0005 for all the layers. Do I need to reduce the weight_decay of last fc layer using fc7_wd_mult? Will it lead to overfitting? Thanks.

**Comment by nttstar** [comment]
No converge problem on 1M ID dataset from my experiments.

**Comment by forseeking** [comment]
Thanks for your reply. 

> No converge problem on 1M ID dataset from my experiments.

Thanks for your reply. I am now using the softmax loss only. In this case do I need to reduce the weight decay of the last fc layer?
